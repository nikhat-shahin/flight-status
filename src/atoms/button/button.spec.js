import React from 'react';
import {
    shallow,
    configure
} from 'enzyme';
import Adapter from "enzyme-adapter-react-16";
import {
    createSerializer
} from 'enzyme-to-json';
import toJson from 'enzyme-to-json';
import Button from './index';

configure({
    adapter: new Adapter()
});
expect.addSnapshotSerializer(createSerializer({
    mode: 'deep'
}));

it('should render Button correctly', () => {
    const component = shallow( <
        Button title = {
            `Test Title`
        }
        handleClick = {
            jest.fn()
        }
        />,
    );
    expect(toJson(component)).toMatchSnapshot();
});
