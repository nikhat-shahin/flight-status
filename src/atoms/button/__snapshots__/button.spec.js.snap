// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Button correctly 1`] = `
<button
  className="button undefined_button"
  onClick={[MockFunction]}
>
   
  Test Title
   
</button>
`;
